iquip/apps/builder.py

```diff
@@ -574,6 +582,110 @@ def scanArguments(self) -> Dict[str, Any]:
         }
 
 
+class _CenterScan(_BaseScan):
+    """Widget for center scan in _ScanEntry.
+
+    Attributes:
+        centerSpinBox: QDoubleSpinBox for center argument inside state.
+        spanSpinBox: QDoubleSpinBox for span argument inside state.
+        stepSpinBox: QDoubleSpinBox for step argument inside state.
+        randomizeCheckBox: QCheckBox for randomize argument inside state.
+    """
+    def __init__(
+        self,
+        procdesc: Dict[str, Any],
+        state: Dict[str, Any],
+        parent: Optional[QWidget] = None
+        ):
+        """Extended.
+
+        Args:
+            state: Each key and its value are:
+              center: The center point for the CenterScan sequence.
+              span: The length of the RangeScan sequence.
+              step: The size of step between each number at the RangeScan sequence.
+              randomize: The boolean value that decides whether to shuffle the CenterScan sequence.
+        """
+        super().__init__(procdesc, parent=parent)
+        self.centerSpinBox = QDoubleSpinBox(self)
+        self.applyProperties(self.centerSpinBox)
+        self.centerSpinBox.setValue(state["center"] / self.scale)
+        self.spanSpinBox = QDoubleSpinBox(self)
+        self.applyProperties(self.spanSpinBox)
+        self.spanSpinBox.setValue(state["span"] / self.scale)
+        self.stepSpinBox = QDoubleSpinBox(self)
+        self.applyProperties(self.stepSpinBox)
+        self.stepSpinBox.setValue(state["step"] / self.scale)
+        self.randomizeCheckBox = QCheckBox("Randomize", self)
+        self.randomizeCheckBox.setChecked(state["randomize"])
+        # layout
+        self.layout.addWidget(QLabel("center:", self), 0, 0)
+        self.layout.addWidget(self.centerSpinBox, 0, 1)
+        self.layout.addWidget(QLabel("span:", self), 1, 0)
+        self.layout.addWidget(self.spanSpinBox, 1, 1)
+        self.layout.addWidget(QLabel("step:", self), 2, 0)
+        self.layout.addWidget(self.stepSpinBox, 2, 1)
+        self.layout.addWidget(self.randomizeCheckBox, 3, 1)
+
+    def scanArguments(self) -> Dict[str, Any]:
+        """Overridden.
+        
+        Returns the arguments of the center scan.
```

Comment:
I'm sorry for the late review, but this line looks unnecessary because there is an sufficient explanation in `_BaseScan.scanArguments()`.
So please leave only "Overridden" for all scan classes.